tests/unit/modules/network/ovs/test_openvswitch_bond.py

```diff
@@ -0,0 +1,176 @@
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# Make coding more python3-ish
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+from ansible_collections.openvswitch.openvswitch.tests.unit.compat.mock import (
+    patch,
+)
+from ansible_collections.openvswitch.openvswitch.plugins.modules import (
+    openvswitch_bond,
+)
+from ansible_collections.openvswitch.openvswitch.tests.unit.modules.utils import (
+    set_module_args,
+)
+from .ovs_module import TestOpenVSwitchModule, load_fixture
+
+test_name_side_effect_matrix = {
+    "test_openvswitch_bond_absent_idempotent": [(0, "", "")],
+    "test_openvswitch_bond_absent_removes_bond": [
+        (0, "list_ports_bond_br.cfg", ""),
+        (0, "get_port_bond0_other_config.cfg", ""),
+        (0, "get_port_bond0_external_ids.cfg", ""),
+        (0, "", ""),
+        (0, "", ""),
+    ],
+    "test_openvswitch_bond_present_idempotent": [
+        (0, "list_ports_bond_br.cfg", ""),
+        (0, "get_port_bond0_other_config.cfg", ""),
+        (0, "get_port_bond0_external_ids.cfg", ""),
+        (0, "", ""),
+        (0, "", ""),
+    ],
+    "test_openvswitch_bond_present_creates_bond": [
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+    ],
+    "test_openvswitch_bond_present_creates_lacp_bond": [
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+        (0, "", ""),
+    ],
+}
+
+
+class TestOpenVSwitchBondModule(TestOpenVSwitchModule):
+
+    module = openvswitch_bond
+
+    def setUp(self):
+        super(TestOpenVSwitchBondModule, self).setUp()
+
+        self.mock_run_command = patch(
+            "ansible.module_utils.basic.AnsibleModule.run_command"
+        )
+        self.run_command = self.mock_run_command.start()
+        self.mock_get_bin_path = patch(
+            "ansible.module_utils.basic.AnsibleModule.get_bin_path"
+        )
+        self.get_bin_path = self.mock_get_bin_path.start()
+
+    def tearDown(self):
+        super(TestOpenVSwitchBondModule, self).tearDown()
+
+        self.mock_run_command.stop()
+        self.mock_get_bin_path.stop()
+
+    def load_fixtures(self, test_name):
+        test_side_effects = []
+        for s in test_name_side_effect_matrix[test_name]:
+            rc = s[0]
+            out = s[1] if s[1] == "" else str(load_fixture(s[1]))
+            err = s[2]
+            side_effect_with_fixture_loaded = (rc, out, err)
+            test_side_effects.append(side_effect_with_fixture_loaded)
+        self.run_command.side_effect = test_side_effects
+
+        self.get_bin_path.return_value = "/usr/bin/ovs-vsctl"
+
+    def test_openvswitch_bond_absent_idempotent(self):
+        set_module_args(dict(state="absent", bridge="bond-br", port="bond0"))
+        self.execute_module(
+            test_name="test_openvswitch_bond_absent_idempotent"
```

Comment:
I'm assuming this is supposed to result in no commands issued, you can ensure that with `commands=[]` (vs not specifying `commands`, which makes no assumptions about what commands might be returned)
```suggestion
            commands=[],
            test_name="test_openvswitch_bond_absent_idempotent"
```

Reply:
That's correct, no commands are expected in these cases. Thanks for the assist!